pkg/router/f5/f5.go

```diff
@@ -1187,7 +1243,7 @@ func (f5 *f5LTM) getRoutes(policyname string) (map[string]bool, error) {
 	}
 
 	url := fmt.Sprintf("https://%s/mgmt/tm/ltm/policy/%s/rules",
-		f5.host, policyname)
+		f5.host, f5.iControlUriResourceId(policyname))
```

Comment:
Each instance of the controller pod will run with only one given partition. All internal data structures need not bother about multiple partitions.